Documentation/configuration/index.rst

```diff
@@ -19,3 +19,35 @@ Core Agent
    sctp
    vlan-802.1q
    argocd-issues
+
+Verify Signed Container Images
```

Comment:
Please let's move this section to a dedicated file, and only add a link to it in the `index.rst`.